WalletWasabi.Fluent/Views/Wallets/Send/SendFeeView.axaml

```diff
@@ -180,7 +180,7 @@
             <Setter Property="AreaMinViableWidth" Value="40"/>
             <Setter Property="AreaMinViableHeight" Value="20"/>
             <Setter Property="Focusable" Value="True"/>
-            <Setter Property="YAxisTitle" Value="Increasing Cost (sats/vbyte)"/>
+            <Setter Property="YAxisTitle" Value="Increasing Cost (sat/vByte)"/>
```

Comment:
meh, whatever 😄

Reply:
I changed it just for the sake of consistency.